__tests__/Autocomplete.test.js

```diff
@@ -395,4 +395,42 @@ describe('Autocomplete onChange simulations', () => {
     expect(component.state.selectedOptions[0]).toBe(ABC_SORTED_MNEMONICS[3]);
     expect(component.state.isOpen).toBe(false);
   });
+
```

Comment:
@DominikGuzei 
I was on a short flight and had nothing to do so this kept me busy. I didn't think about this test until today, but I feel it covers an important piece of Autocomplete's functionality.